scripts/airgapped/prerequisites.sh

```diff
@@ -9,8 +9,9 @@ sudo apt update
 
 echo "Installing Docker"
 sudo snap install docker
-sudo groupadd docker
-sudo usermod -aG docker $USER
+sudo addgroup --system docker
+sudo adduser $USER docker
+newgrp docker
```

Comment:
This is an interesting one. The reason we need this command is for the shell execution environment to pickup the group changes (that $USER is now in the `docker` group). But by doing what happens is:
1. Our script was running normally
2. In this line it essentially opens up a new bash session
3. The new session doesn't run any code
4. Once we Ctrl^D then the session closes and the execution of the script continues

The above means that for the whole script to finish then a user needs to manuall Ctrl^D for all the commands to run. Which we don't want since it's confusing if the script has completed or not

I suggest we do the following:
1. We don't run `newgrp docker` so the script finishes in one go
2. We update the `README.md` and tell users they need to reboot their machine after the prerequisites script is run 
3. We update the `tests/airgapped/setup/setup.sh` to echo at the end that users need to reboot their machines, for the docker changes
4. We add a small comment in this file to denote why we don't do `newgrp docker`

This way the script completes in one go,

Reply:
updated